scripts/build_example

```diff
@@ -41,28 +41,46 @@ else
     TARGET="$NAME"
 fi
 
+bazel_build_flags+=(
+    '--symlink_prefix=bazel-client-'
+    "--compilation_mode=$COMPILATION_MODE"
+)
+
 readonly RUST_FOLDER="$(find examples/"$TARGET" -type d -name rust)"
 if [[ -n "$RUST_FOLDER" ]]; then
     for module in examples/"$TARGET"/module*/rust/Cargo.toml; do
-    cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
+        cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
     done
-fi
 
-bazel_build_flags+=(
-    '--symlink_prefix=bazel-client-'
-    "--compilation_mode=$COMPILATION_MODE"
-)
+    # Serialize application configuration for Rust module.
+    if [[ "hello_world" != "$TARGET" ]]; then
```

Comment:
Could you invert this condition, instead of having to consider a double negative in the else case? Also below in other scripts.

Reply:
Done